.travis.yml

```diff
@@ -26,13 +30,19 @@ matrix:
 
 before_install:
   - npm config set spin false
-  - npm install -g bower phantomjs-prebuilt
+  - npm install -g bower phantomjs-prebuilt yarn
```

Comment:
Yarn recommends to install it from a script instead from npm. Apparently there was some edge cases with it. https://yarnpkg.com/lang/en/docs/install-ci/#travis-tab